ereuse_devicehub/resources/device/models.py

```diff
@@ -881,3 +881,59 @@ def add_all_to_session(cls, session: db.Session):
                 'COPY common.manufacturer FROM STDIN (FORMAT csv)',
                 f
             )
+
+
+
+update_timestamp(Computer)
+update_timestamp(Desktop)
+update_timestamp(Laptop)
+update_timestamp(Server)
+update_timestamp(Monitor)
+update_timestamp(ComputerMonitor)
+update_timestamp(TelevisionSet)
+update_timestamp(Projector)
+update_timestamp(Mobile)
+update_timestamp(Smartphone)
+update_timestamp(Tablet)
+update_timestamp(Cellphone)
+update_timestamp(Component)
+update_timestamp(GraphicCard)
+update_timestamp(DataStorage)
+update_timestamp(HardDrive)
+update_timestamp(SolidStateDrive)
+update_timestamp(Motherboard)
+update_timestamp(NetworkAdapter)
+update_timestamp(Processor)
+update_timestamp(RamModule)
+update_timestamp(SoundCard)
+update_timestamp(Display)
+update_timestamp(Battery)
+update_timestamp(Camera)
+update_timestamp(ComputerAccessory)
+update_timestamp(SAI)
+update_timestamp(Keyboard)
+update_timestamp(Mouse)
+update_timestamp(MemoryCardReader)
+update_timestamp(Networking)
+update_timestamp(Router)
+update_timestamp(Switch)
+update_timestamp(Hub)
+update_timestamp(WirelessAccessPoint)
+update_timestamp(Printer)
+update_timestamp(LabelPrinter)
+update_timestamp(Sound)
+update_timestamp(Microphone)
+update_timestamp(Video)
+update_timestamp(VideoScaler)
+update_timestamp(Videoconference)
+update_timestamp(Cooking)
+update_timestamp(Mixer)
+update_timestamp(DIYAndGardening)
+update_timestamp(Drill)
+update_timestamp(PackOfScrewdrivers)
+update_timestamp(Home)
+update_timestamp(Dehumidifier)
+update_timestamp(Stairs)
+update_timestamp(Recreation)
+update_timestamp(Bike)
+update_timestamp(Racket)
```

Comment:
I guess that will be a more maintanable way to do that. Please refactor.

Reply:
Looking at `ereuse_devicehub/resources/device/models.py` I see that models follow an inheritance tree. Could a signal be defined on the root class of the inheritance and then all the other models will inherit the linked signal?

In fact, as `updated` is an attribute defined on `class Device` (Base class for any type of physical object that can be identified.); I guess that Flask will provide a way to have a timestamp automatically updated when the object changes, doesn't it?

Reply:
Refactory done with a propagate var in listen function